frontend/src/pages/Alerts/Alerts.tsx

```diff
@@ -392,6 +398,38 @@ function AlertsPageLoaded({
 								<NewAlertMenu />
 							)}
 						</Box>
+						<Callout
+							title="Want to learn more about
+												Alerts?"
```

Comment:
the callout looks good, but should we have some way to close the callout if you've watched the video? maybe we can save the closed state in local storage so that if you close it once, we won't show you this card again. i can see folks wanting to hide it after looking at the docs since it takes up a lot of vertical space on the page

Reply:
I can do a local storage val for this with the "X" -> I original was doing this but wasn't sure if it was a code smell to use local storage

Reply:
Added